cmd/fossa/main.go

```diff
@@ -328,21 +246,27 @@ func defaultCmd(c *cli.Context) {
 			mainLogger.Fatalf("Could marshal analysis results: %s", err.Error())
 		}
 		fmt.Println(string(buildData))
+		return
 	}
 
-	if config.analyzeConfig.noUpload {
-		return
+	s.Stop()
+	s.Suffix = fmt.Sprintf(" Writing configuration...")
```

Comment:
Why does this happen at the end?

Reply:
We only want to write configuration if the defaultCmd was successful, esp useful in combination with `--force`, where we only want to re-write the configuration if successful.